Update disability content (#4649)
DFE-Digital/get-into-teaching-app: app/views/content/funding-and-support/if-youre-disabled.md

---

> title: "Support if you have a long-term health condition, learning difficulty, or neurodiversity"

Author: ```suggestion
title: "Support if you have a health condition, learning difficulty, or neurodiversity"
```

heading: "Funding and support if you have a health condition, learning difficulty, or neurodiversity"
subcategory: Extra support
description: |-

>     Find out about the support you can get while training to teach if you're neurodivergent, have a long-term health condition, or other accessibility need.

Author: ```suggestion
    Find out about the support you can get while training to teach if you're neurodivergent, have a long-term health condition, or other accessibility needs.
```

heading: "Funding and support if you're disabled"
subcategory: Extra support
description: |-

>     Find out about the support you can get while training to teach if you're neurodivergent, have a long-term physical or mental health condition, or have any other accessibility need.

Reviewer: This is a bit long for a meta description - can it be shortened at all?

    text: "Growing up as a profoundly deaf child, I had a positive upbringing in my mainstream primary school. The teachers were amazing in terms of support and treating me equally. They inspired me to become a primary school teacher."
    name: "Habiba, primary school teacher"
    classes: quote--indent
cta_arrow_link:
  dsa:
    link_target: "https://www.gov.uk/disabled-students-allowance-dsa"

>     link_text: "Find out if you're eligible for a bursary or scholarship"

Reviewer: This isn't the right link text!

  access-to-work:
    link_target: "https://www.gov.uk/access-to-work"
    link_text: "Find out more about Access to Work"
  adjustments:
    link_target: "https://www.gov.uk/access-to-work"

>     link_text: "Find out more about adjustments"

Reviewer: ```suggestion
    link_text: "Find out more about adjustments to help you apply and train"
```
Worth adding more context?

Also - presume this will go to a new page in Train to be a teacher (when added to PR!)?

### Disabled Students' Allowance (DSA)

If you’re on a fee-paying teacher training course, you may be eligible for Disabled Students’ Allowance (DSA). You cannot get DSA if your course pays a salary. 

DSA can help towards covering any study-related costs you have because of a mental health problem, long term illness or any other disability. You can get DSA on its own or in addition to other student finance. 

It can take up to 14 weeks to get your DSA, so you should take this into consideration if you need the support in place for the start of your course.

$dsa$

### Access to Work

If you get paid for your training, for example, salaried training or a teacher degree apprenticeship, you may be eligible for support through Access to Work. 

Through Access to Work, you can apply for: 

* a grant to help pay for practical support with your work 
* support with managing your mental health at work

Examples could include paying towards accessible transport to get you to school or help with funding specialist equipment.  

> Access to Work can also pay for a workplace assessment to see what you need at work. 

Reviewer 2: Maybe add ..."in order to do your job"? Or maybe not, just feels like something is missing from this sentence

Author: ```suggestion
Access to Work can also pay for a workplace assessment to see what you need in order to do your job. 
```

> You can apply for Access to Work up to 12 weeks before you start paid work. You should apply as soon as possible if you need the support in place before you start your training.

Reviewer 2: Bit confused about this bit as it talks about starting paid work and then about training?

Reviewer: ```suggestion
You can apply for Access to Work up to 12 weeks before you start your salaried training or apprenticeship. You should apply as soon as possible if you need the support in place before you start.
```
Would this be clearer?

> You can apply for Access to Work up to 12 weeks before you start paid work. You should apply as soon as possible if you need the support in place before you start your training.

Author: ```suggestion
You can apply for Access to Work up to 12 weeks before you start your training. You should apply as soon as possible if you need the support in place before you start.
```

### Disabled Students' Allowance (DSA)

If you’re on a fee-paying teacher training course, you may be eligible for Disabled Students’ Allowance (DSA). You cannot get DSA if your course pays a salary. 

> DSA can help towards covering any study-related costs you have because of a mental health problem, long term illness or any other disability. You can get DSA on its own or in addition to other student finance. 

Reviewer: ```suggestion
DSA can help cover any study-related costs you have because of a mental health problem, long-term illness or any other disability. You can get DSA on its own or in addition to other student finance. 
```

It can take up to 14 weeks to get your DSA, so you should take this into consideration if you need the support in place for the start of your course.

$dsa$

> You could also receive a tax-free bursary or scholarship of up to $scholarships_generic_max$ to train to teach certain subjects. [Find out more about your eligibility for a scholarship or bursary](/funding-and-support/scholarships-and-bursaries).

Reviewer 2: ```suggestion
You could also receive a tax-free bursary or scholarship of up to $scholarships_generic_max$ to train to teach certain subjects. Find out more about [your eligibility for a scholarship or bursary](/funding-and-support/scholarships-and-bursaries).
```

Reviewer 2: Just a suggestion as looks a bit weird being a full link as part of a para

### Access to Work

If you get paid for your training, for example, salaried training or a teacher degree apprenticeship, you may be eligible for support through Access to Work. 

Through Access to Work, you can apply for: 

* a grant to help pay for practical support with your work 
* support with managing your mental health at work

Examples could include paying towards accessible transport to get you to school or help with funding specialist equipment.  

Access to Work can also pay for a workplace assessment to see what you need in order to do your job. 

You can apply for Access to Work up to 12 weeks before you start your salaried training or apprenticeship. You should apply as soon as possible if you need the support in place before you start.

$access-to-work$

## Adjustments to help you train

Teacher training providers can make lots of adjustments so you can attend an interview and complete your teacher training.

$adjustments$

> ## Your rights

Reviewer: Would this section fit better on the page in Train to be a teacher?
